website/docs/r/disk_encryption_set.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_disk_encryption_set"
+sidebar_current: "docs-azurerm-resource-disk-encryption-set"
+description: |-
+  Manage Azure DiskEncryptionSet instance.
+---
+
+# azurerm_disk_encryption_set
+
+Manage an Azure DiskEncryptionSet instance.
+
+-> **NOTE:** The DiskEncryptionSet service is currently in public preview and are only available in a limited set of regions. 
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "des-example-rg"

```

Comment:
we use "example-resources" for all new examples

```suggestion
  name     = "example-resources"
```

Reply:
Done